api/src/main/java/org/openmrs/api/handler/PatientDataUnvoidHandler.java

```diff
@@ -72,6 +73,11 @@ public void handle(Patient patient, User originalVoidingUser, Date origParentVoi
 					}
 				}
 			}
+
+			if (!patient.getVoided()) {
```

Comment:
I'd think the if clause is unnecessary, the is only called when the patient is getting unvoided so they have to be unvoided anyways